### PR TITLE
[WEBSITE] Refactor Press View to display data from the database

### PR DESCRIPTION
**Pull request by OchiengPaul442**
#### Summary of Changes
This PR includes several updates to improve the Press page.
- The Press page view has been refactored to display data from the backend.
- The get_publisher_logo method in serializer.py has been refactored to fix an “Undefined” error.
- The Press.scss file has been updated to accommodate the large card.
- Included show more or show less button, that appears once the number of articles exceeds 5.
- Additional changes have been made to ensure that the website stack runs smoothly on Windows OS.

#### Status of maturity (all need to be checked before merging):

- [x] I've tested this locally
- [x] I consider this code done

#### How should this be manually tested?

- [Jira_card_number]() [(If exists)](https://airqoteam.atlassian.net/browse/WEB-404)

#### Screenshots (optional)
image

### CODE REVIEW

**Review by Baalmart — APPROVED** [review]
Thanks @OchiengPaul442 , looks good to me.
Please follow up with @FaithDaka on the side for the final review and then we shall see only merge after her approval.

File: website/frontend/reduxStore/Press/PressSlice.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+export const loadPressData = () => async (dispatch) => {
+  try {
+    dispatch(startLoading());
+    const resData = await getAllPressApi();
+    dispatch(loadPressSuccess(resData));
+  } catch (err) {
+    dispatch(loadPressFailure(err.message));
+  }
+};
```
Comment (FaithDaka) [inline website/frontend/reduxStore/Press/PressSlice.js:41]: Great work with refactoring @OchiengPaul442 

Given that Javascript is an interpreted language, it's always good practice to have logic functions at the top of the page such that nothing breaks later on. I'll have this refactored then clear the PR for merge

**Comment by FaithDaka** [comment]
@Baalmart Ready for merge

File: website/frontend/reduxStore/Press/PressSlice.js (hunk 1 above)
Comment (OchiengPaul442) [reply website/frontend/reduxStore/Press/PressSlice.js]: Okay, @FaithDaka.

**Comment by Baalmart** [comment]
> @Baalmart Ready for merge

thanks @FaithDaka

**Pull request merged**